openmetadata-service/src/main/java/org/openmetadata/service/events/subscription/slack/SlackWebhookEventPublisher.java

```diff
@@ -7,29 +9,29 @@
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
 import lombok.extern.slf4j.Slf4j;
-import org.openmetadata.schema.entity.alerts.Alert;
-import org.openmetadata.schema.entity.alerts.AlertAction;
+import org.openmetadata.schema.entity.events.EventSubscription;
 import org.openmetadata.schema.type.ChangeEvent;
 import org.openmetadata.schema.type.Webhook;
-import org.openmetadata.service.alerts.AlertsActionPublisher;
 import org.openmetadata.service.events.errors.EventPublisherException;
-import org.openmetadata.service.resources.events.EventResource;
+import org.openmetadata.service.events.subscription.SubscriptionPublisher;
+import org.openmetadata.service.jdbi3.CollectionDAO;
+import org.openmetadata.service.resources.events.ChangeEventResource;
 import org.openmetadata.service.util.ChangeEventParser;
 import org.openmetadata.service.util.JsonUtils;
 
 @Slf4j
-public class SlackWebhookEventPublisher extends AlertsActionPublisher {
+public class SlackWebhookEventPublisher extends SubscriptionPublisher {
```

Comment:
SlackWebhookEventPublisher to SlackEventPublisher or SlackPublisher

Reply:
done